api/services/user.ts

```
@@ -0,0 +1,28 @@
+import bcrypt from 'bcryptjs';
+
+import bcrypt from 'bcryptjs';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'bcrypt'. Consider to delete it or rename it.

'bcrypt' is defined here:


(lint/suspicious/noRedeclare)